### ISSUE TITLE
Assertion "CheckSizerFlags(!((flags) & (wxALIGN_CENTRE_VERTICAL)))" failed

### ISSUE THREAD

**Issue by ahall41** [larspalo/GOODF#69]
Running on Ubuntu Cinnamon:

```
/src/common/sizer.cpp(2258): assert "CheckSizerFlags(!((flags) & (wxALIGN_CENTRE_VERTICAL)))" failed in DoInsert(): wxALIGN_CENTRE_VERTICAL will be ignored in this sizer: only horizontal alignment flags can be used in vertical sizers

DO NOT PANIC !!

If you're an end user running a program not developed by you, please ignore this message, it is harmless, and please try reporting the problem to the program developers.

You may also set WXSUPPRESS_SIZER_FLAGS_CHECK environment variable to suppress all such checks when running this program.

If you're the developer, simply remove this flag from your code to avoid getting this message. You can also call wxSizerFlags::DisableConsistencyChecks() to globally disable all such checks, but this is strongly not recommended.

```
Reporting the message to the developer as suggested :-)

**Comment by larspalo** [comment]
Thanks for reporting! Yes, I remember seeing this warning when building with wx 3.2 a while ago. It's harmless but eventually needs fixing.

**Comment by larspalo** [comment]
Should be fixed with 97a0c5f5babc97b27b49c22ed31f5d9e963896b1.

**Comment by ahall41** [comment]
:-)
